docs/connectors/kubernetes.md

```diff
@@ -72,6 +75,15 @@ This command will remove the `admin` role, granted in the previous example, from
 infra grants remove ops@example.com cluster.namespace --role cluster-admin
 ```
 
+### Custom Kubernetes Roles
+
+If the provided roles are not sufficient, additional roles can be configured to integrate with Infra. To add a new role, create a ClusterRole and add label `app.infrahq.com/include-role=true`.
```

Comment:
The suggestion isn't completely accurate because the user can create a entirely new ClusterRole. The following sentence clarifies it further by providing more details

Reply:
Ah nvm I see the confusion. Add more details